roles/ansible/tower/config-ansible-tower/defaults/main.yml

```diff
@@ -8,6 +8,9 @@ ansible_tower_version: 3.3.0-1
 # oc clients found at 'https://mirror.openshift.com/pub/openshift-v3/clients/'
 ansible_tower_oc_download_url: https://mirror.openshift.com/pub/openshift-v3/clients/3.10.47/linux/oc.tar.gz
 
+# helm clients found at 'https://github.com/helm/helm/releases'
+ansible_tower_helm_download_url: ""
```

Comment:
if we don't have a default value for this, I'd suggest leaving it out ... preferably we'd have a default though.